obd_ontology/extract_dtcs_and_components_from_excel_file.py

```diff
@@ -0,0 +1,131 @@
+import argparse
```

Comment:
Sounds good, but there is one problem: I want the "valid_special_characters" variable to be consistent across app.py and extract_dtcs_and_components_from_excel_file.py. Right now it is defined in extract_dtcs_and_components_from_excel_file.py and imported in app.py. This won't be possible when the script is removed from obd_ontology. I also don't like importing things from app.py, because there are things that happen in app.py outside any function and it could possibly cause side effects. Where would you place this "valid_special_characters" variable, so that it can be used by both scripts?

Reply:
At the moment, I'd suggest `config.py`. It also makes sense, you can configure the characters that you consider valid. Later it may be part of a `util.py`.